### PR TITLE
Om feature display nav bar

### LINKED ISSUE #23: Display NavBar
A functional component that will render at the top of the application

Screen Shot 2021-01-06 at 4 21 11 PM

### PR DESCRIPTION
**Pull request by Atos20**
### What does this PR do?
- Creates NavBar components 
- Styles NavBar compoenens
### Where should the reviewer start?

- src/components/NavBar/NavBar.css
- src/components/NavBar/NavBar.tsx

### How should this be manually tested?
Run `npm start and`
### Questions:
closes #23 
### Screenshots (if appropriate)
Screen Shot 2021-01-06 at 4 19 11 PM

### CODE REVIEW

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <div className='nav-container'>
+      <div className="inner-menu">
+        <ImMenu className='icon'/>
+        <h1 className='menu-title'>menu</h1>
+      </div>
+    </div>
+  )
+}
+
```
Comment (timkeresey) [inline src/components/NavBar/NavBar.tsx:17]: Looks like we just need to add the links. Awesome, Orlando!

**Review by timkeresey — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.1.0",
```
Comment (priyapower) [inline package.json:16]: What are react icons?

File: src/apiCalls.tsx [hunk 3]
```diff
@@ -0,0 +1 @@
+export{}
```
Comment (priyapower) [inline src/apiCalls.tsx:1]: I believe this file sets up for your api calls to the be server or external apis

File: src/components/App/App.css [hunk 4]
```diff
@@ -1,38 +1,5 @@
...
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+.name{
```
Comment (priyapower) [inline src/components/App/App.css:1]: Css file for the App.tsx file

File: src/components/Error/Error.tsx [hunk 5]
```diff
@@ -4,7 +4,11 @@ import React, { useState, useEffect } from 'react';
 const Error: React.FC = () => {
 
   return (
-
+    <>
```
Comment (priyapower) [inline src/components/Error/Error.tsx:7]: I believe this handles the error pathways

File: src/components/NavBar/NavBar.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
+@import '../../_variables.css';
```
Comment (priyapower) [inline src/components/NavBar/NavBar.css:1]: Nav Css for Nav.tsx

File: src/components/NavBar/NavBar.tsx [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```
Comment (priyapower) [inline src/components/NavBar/NavBar.tsx:1]: I believe this handles the navigation bar functionality

File: src/components/Sourvey/Sourvey.tsx [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```
Comment (priyapower) [inline src/components/Sourvey/Sourvey.tsx:1]: I believe this handles the Survey page functionality

File: src/index.tsx [hunk 9]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import './_variables.css';
-import App from './App';
+import App from './components/App/App';
```
Comment (priyapower) [inline src/index.tsx:5]: I believe this renders the application, the "runner" file

File: src/components/App/App.tsx [hunk 10]
```diff
@@ -3,36 +3,42 @@ import React, { useState, useEffect } from 'react';
 //types
 import { data } from '../../types';
 //components
-import Home from '../Home/Home';
-import Error from '../Error/Error';
-import Journey from '../Journey/Journey';
+// import Home from '../Home/Home';
```
Comment (priyapower) [inline src/components/App/App.tsx:6]: I believe this handles the welcome page functionality

**Review by priyapower — COMMENTED** [review]
Great job! I had a couple questions for my own understanding. Please answer at your leisure

**Pull request merged**